packages/compiler/server/serverlib.ts

```diff
@@ -606,6 +613,36 @@ export function createServer(host: ServerHost): Server {
     }
   }
 
+  async function getDocument(type: Type): Promise<string> {
+    return type.kind;
+  }
+
+  async function getTypeDetails(params: HoverParams): Promise<Hover> {
```

Comment:
That function should still be `getHover` or `onHover`. Saying there should be another method called `getTypeDetails` that takes a `type: Type` and return the documentation